persist/log.go

```diff
@@ -0,0 +1,30 @@
+package persist
+
+import (
+	"log"
+	"os"
+)
+
+// FileLogger is a wrapper for the standard library logger that enforces logging
+// into a file with the Sia-standard settings.
```

Comment:
not quite. It also establishes the format that gets used, including choosing to display all timestamps in UTC.